README.md

```diff
@@ -1,2 +1,56 @@
 # schoolLibrary
 A school library app to record what books are in the library and who borrows them. The app that you will create will allow you to: add new students or teachers, add new books and save records of who borrowed a given book and when.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+**Prerequisites**
+To get started ensure that you have the following:
+
+> Run the command(s) in terminal to ensure that Ruby is installed on your system
+
+```bash
+$ ruby --version
+$ irb
+```
+
+### Setup
+
+**Cloning** ⬇️
+To clone the project:
+
+```bash
+$ git clone https://github.com/Estherstarr/school-library.git
```

Comment:
[OPTIONAL] Your README has the major headings. But kindly make your README more descriptive. I wouldn't know how to use your app without prior knowledge. You can have a look at this [PR](https://github.com/fobadara/quest-student-library/blob/development/README.md) for some hints.